ui/src/components/MyPollsPreview/MyPollsPreview.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { render } from 'react-dom';
+import './MyPollsPreview.css';
+
+class MyPollsPreview extends React.Component {
+    render() {
+        const isFinished = (this.props.pollId != -1);
+
+        // default unfinished draft
+        var backgroundColor = '#62B0FF';
```

Comment:
We can modify this when we'll attach middleware, but essentially, you'll want to define CSS styles for draft and active polls and selectively assign styles below by checking the "active" property. I'll explain how when we get to the middleware writing, which will be very soon.